package.json

```diff
@@ -1,47 +1,33 @@
 {
-  "name": "ditto-bnb",
-  "version": "0.1.0",
+  "name": "ditto-react-demo",
   "private": true,
-  "dependencies": {
-    "@dittowords/cli": "3.7.0",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "ditto-react": "1.4.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
-    "sass": "^1.42.1",
-    "web-vitals": "^1.0.1"
-  },
+  "version": "0.0.0",
+  "type": "module",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
     "sync": "cd src && ditto-cli pull"
```

Comment:
Similar to our own internal conventions, I think an even better name for this command would be `ditto:pull` (and that is what @kadenbarlow has done in the Android repo: https://github.com/dittowords/ditto-android-demo/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R11)